ui/app/src/utils/xml.ts

```diff
@@ -145,13 +148,13 @@ export function wrapElementInAuxDocument(element: Element): XMLDocument {
   return fromString(`<?xml version="1.0" encoding="UTF-8"?>${element.outerHTML}`);
 }
 
-export function toJS(xml: string): any;
-export function toJS(xml: XMLDocument): any;
-export function toJS(xml: string | XMLDocument): any {
+export function toJS(xml: string, options?: X2jOptionsOptional): any;
+export function toJS(xml: XMLDocument, options?: X2jOptionsOptional): any;
+export function toJS(xml: string | XMLDocument, options?: X2jOptionsOptional): any {
   if (typeof xml !== 'string') {
     xml = serialize(xml);
   }
-  return parser.parse(xml, {});
+  return parser.parse(xml, options);
```

Comment:
Default these to the ones you had.
{
  blah: blah,
  blah: blah,
  ...options
}